doc/chapters/gateways/10-dbus-gateway.rst

```diff
@@ -7,8 +7,9 @@ interfaces.
 The gateway will create a socket for the D-Bus connection being proxied.
 The socket will be placed in a directory accessible from within the
 container, and the applications running inside the container are expected
-to use this socket when communicating with the outside D-Bus system. The gateway uses an external
-program ``dbus-proxy`` for creation and communication over the proxied bus connection.
+to use this socket when communicating with the outside D-Bus system.
+The gateway uses ``dbus-proxy``, which is included in SoftwareContainer as a submodule,
+for creation and communication over the proxied bus connection.
 
```

Comment:
I think it's a good idea to point out that it's a submodule, but we now lose the point that it is a separate program, i.e. not part of SC that does this. Try to mention both these things.